R/PackageInheritanceReporter.R

```diff
@@ -212,7 +212,7 @@ InheritanceReporter <- R6::R6Class(
 
         plotNodeColorScheme = list(
             field = "classType"
-            , palette = c('#a6cee3', '#1f78b4', '#b2df8a')
+            , palette = c('#f0f9e8', '#bae4bc', '#7bccc4')
```

Comment:
Is this colorblind assessable?

Reply:
I got them from ColorBrewer with the option on. 

Looks good:
<img width="500" alt="screen shot 2018-12-04 at 10 48 01 pm" src="https://user-images.githubusercontent.com/2721979/49490684-b7b71c80-f816-11e8-8f0f-5ad55fe49810.png">